electrum/base_wizard.py

```diff
@@ -368,9 +371,14 @@ def f(derivation, script_type):
                 self.run('on_hw_derivation', name, device_info, derivation, script_type)
             self.derivation_and_script_type_dialog(f)
         elif purpose == HWD_SETUP_DECRYPT_WALLET:
-            derivation = get_derivation_used_for_hw_device_encryption()
-            xpub = self.plugin.get_xpub(device_info.device.id_, derivation, 'standard', self)
-            password = keystore.Xpub.get_pubkey_from_xpub(xpub, ()).hex()
+            # The BitBox02 does not support non-standard xpubs
+            if device_info.label == "BitBox02":
+                xpub = self.plugin.get_xpub(device_info.device.id_, "m/84'/1'/0'", "p2wpkh", self)
+                password = hmac_oneshot(b"electrum_password_lulz", bitcoin.DecodeBase58Check(xpub), hashlib.sha512).hex()
```

Comment:
i'd prefer a slightly more serious hmac key ;) this might have to be supported basically forever..

Reply:
`b"serious_electrum_password_salt"` it is :P